publiccode.yml

```
@@ -0,0 +1,37 @@
+publiccodeYmlVersion: "1.0"
```

```suggestion
publiccodeYmlVersion: "0.2"
```
J'ai l'impression que l'adhésion est plutôt à la version 0.2 chez code.gouv

---

la version 1.0 n'existe pas encore